src/java.base/share/classes/javax/crypto/CryptoPermission.java

```diff
@@ -449,8 +448,8 @@ private boolean equalObjects(Object obj1, Object obj2) {
 }
 
 /**
- * A CryptoPermissionCollection stores a set of CryptoPermission
- * permissions.
+ * A {@code CryptoPermissionCollection} stores a set of
+ * {@code CryptoPermission} permissions.
```

Comment:
Use `objects` instead of `permissions` for consistency's sake?

Reply:
Fixed.